tools/chunk/src/main/scala/org/allenai/nlpstack/chunk/OpenNlpChunker.scala

```diff
@@ -18,6 +18,23 @@ class OpenNlpChunker extends Chunker {
     val chunks = chunker.chunk(tokens.map(_.string).toArray, fixedTokens.map(_.postag).toArray)
     (tokens zip chunks) map { case (token, chunk) => ChunkedToken(token, chunk) }
   }
+
+  //Open NLP runs into random nullPointerExceptions. The following is a workaround that reduces the
+  // probability
+  // of running into errors
+  def chunkPostagged(tokens: Seq[PostaggedToken]): Seq[ChunkedToken] = {
+    var tries = 3
+    while(tries > 0){
+      tries -= 1
+      val ret = try{
+        chunkPostaggedOriginal(tokens)
+      } catch{
+        case e: Throwable => null
+      }
+      if(ret != null)  return ret
+    }
+    return tokens.map{case a: PostaggedToken => ChunkedToken.apply(a,"ERROR")}
```

Comment:
1. I think it would be better to throw an exception than chunk everything with an `"ERROR"` tag.
 2. `return` is very un-scalarific: https://tpolecat.github.io/2014/05/09/return.html. I would use recursion to get the tries to work in a functional manner.